devtools/conda-recipe/meta.yaml

```diff
@@ -28,7 +29,7 @@ requirements:
     - six
     - openmm
     - parmed
-
+    - tqdm
```

Comment:
Ah! I just saw your comment in #135 that it should go away with conda forge, sorry. I'll temporarily deactivate to make tests pass and make a comment in the tests.